fastboot/instance-initializers/content-security-policy.js

```diff
@@ -0,0 +1 @@
+export { default, initialize } from 'ember-cli-content-security-policy/instance-initializers/content-security-policy';
```

Comment:
Can we move the actual implementation of the initializer here instead? Specifically, I'd like to avoid adding the initializer to the build output for users that _aren't_ using fastboot.

In general, I definitely prefer what you've done here (having the actual implementation inside the `addon/` folder), but if we don't move that file here we will force _every_ user to have that extra module...

Reply:
Done in 1fe117fb4adce636c4394c433698eaed8ef3044b.